includes/admin.form.inc

```diff
@@ -65,6 +65,16 @@ function islandora_internet_archive_bookreader_admin_settings_form(array $form,
       '#title' => t('Force the Internet Archive Bookreader into full screen for mobile users?'),
       '#default_value' => variable_get('islandora_internet_archive_bookreader_mobile_full_screen', FALSE),
     ),
+    // XXX: This variable setting exists because Djatoka won't return JP2s
+    // below a filesize that appears to be completely undocumented. Should do
+    // some testing to determine the exact number, but setting at 10kb for now.
```

Comment:
When I said an @XXX I was suggesting to remove the variable completely and note the unconfirmed nature in the code where the magic number is being used.